arch/sim/src/sim/CMakeLists.txt

```diff
@@ -234,6 +234,7 @@ endif()
 target_include_directories(nuttx PRIVATE ${CMAKE_BINARY_DIR}/include/nuttx)
 target_include_directories(nuttx PRIVATE ${CMAKE_CURRENT_LIST_DIR})
 
+target_include_directories(sim_head PUBLIC ${NUTTX_DIR}/sched)
```

Comment:
don't need

Reply:
this include_directories is for target_sources(sim_head PUBLIC sim_head.c sim_doirq.c)